app/modules/onboarding/assets/js/components/Message.js

```diff
@@ -0,0 +1,14 @@
+import react from 'React'
+
+function Message( { tier } ) {
+	const translatedString1 = __('Based on the features you chose, we recommend the ', 'elementor'),
+		translatedString2 = __('plan, or higher', 'elementor');
+
+	return (
+		<>
+			{ `${ translatedString1 }<strong>${ tier }</strong> ${ translatedString2 }` }
+		</>
+	)
+}
+
+export default Message;
```

Comment:
Hey @davseve , 
Played around a bit and I think the prettiest solution will be: 
```
function Message( { tier } ) {
	const translatedString = __( 'Based on the features you chose, we recommend the %s plan, or higher', 'elementor' );
	const [ firstPart, secondPart ] = translatedString.split( '%s' );

	return (
		<>
			{ firstPart }
			<strong>{ __( tier ) }</strong>
			{ secondPart }
		</>
	);
}
```